RecoTauTag/RecoTau/plugins/RecoTauBuilderConePlugin.cc

```diff
@@ -382,6 +456,12 @@ RecoTauBuilderConePlugin::return_type RecoTauBuilderConePlugin::operator()(
   if ( primaryVertexRef.isNonnull() )
     tauPtr->setVertex(primaryVertexRef->position());
 
+  // Set missing tau quantities
+  setTauQuantities(*tauPtr,
```

Comment:
Correct. It is because RecoTauConstructor which was primary coded for RecoTauBuilderCombinatoricPlugin does not sent those quantities for ConeTaus which is built in different way.